CHANGELOG.md

```diff
@@ -78,6 +78,8 @@
   * `-log.rate-limit-enabled`
   * `-log.rate-limit-logs-per-second`
   * `-log.rate-limit-logs-per-second-burst`
+* [ENHANCEMENT] Querier: cancel query requests to ingesters in a zone upon first error received from the zone, to reduce wasted effort spent computing results that won't be used #5764
+* [ENHANCEMENT] Apply dskit HTTPGRPCTracer for more detailed server-side tracing spans and tags on httpgrpc.HTTP/Handle requests #5782
```

Comment:
Perhaps rephrase this to focus on the user impact? eg. "improve tracing of internal HTTP requests sent over httpgrpc"

Reply:
updated in latest commit